### PR TITLE
Various fixes for Lens Flare

### PR DESCRIPTION
**Pull request by skhiat**
Various fixes:
* Default parameters to have far LensFlareSource visible
* Rename doc
* Use the ViewScene parameter 'Flares'
* Change ratio on URP
* minor typo

### CODE REVIEW

**Review by RSlysz — CHANGES_REQUESTED** [review]
Seams good.

Though, can you fix allowed range in property for the component?

File: com.unity.render-pipelines.core/Runtime/PostProcessing/LensFlareComponentSRP.cs [hunk 1]
```diff
@@ -38,12 +38,12 @@ public LensFlareDataSRP lensFlareData
         /// Distance used to scale the Distance Attenuation Curve
         /// </summary>
         [Min(1e-5f)]
-        public float maxAttenuationDistance = 50.0f;
+        public float maxAttenuationDistance = 100.0f;
```
[the remark this answers is not in the document]
Comment (skhiat) [reply com.unity.render-pipelines.core/Runtime/PostProcessing/LensFlareComponentSRP.cs:41]: Here it's not the range, but more about the default values.

**Review by RSlysz — APPROVED** [review]

**Review by Vic-Cooper — APPROVED** [review]
My comma needs are satisfied.

**Comment by sebastienlagarde** [comment]
Added HDRP QA for awareness, sets of small fixes in particular in doc.

**Review by TomasKiniulis — COMMENTED** [review]
Hey @skhiat! Please add a brief explanation what was tested. Also, what does the "Use the ViewScene parameter 'Flares" change do?

**Comment by skhiat** [comment]
> Hey @skhiat! Please add a brief explanation what was tested. Also, what does the "Use the ViewScene parameter 'Flares" change do?

The flare checkbox:
SnRKJux305

The other are fixes related the doc I shared with you.

**Review by TomasKiniulis — APPROVED** [review]

**Pull request merged**